test/test_helper.rb

```diff
@@ -89,6 +89,22 @@ def assert_audited(msg)
       def assert_equal_sets(arr1, arr2)
         assert_equal Set.new(arr1), Set.new(arr2)
       end
+
+      # rubocop:disable Metrics/MethodLength
+      def stub_rbac_permissions(*permissions)
+        role_permissions = permissions.map do |permission|
+          RBACApiClient::Access.new(
+            permission: permission,
+            resource_definitions: nil
+          )
+        end
+        role = RBACApiClient::AccessPagination.new(data: role_permissions)
+        RBACApiClient::AccessApi
+          .any_instance
```

Comment:
You still don't need the any_instance, the client is a constant stored under the class.